.gitignore

```diff
@@ -17,6 +17,7 @@
 .env.development.local
 .env.test.local
 .env.production.local
+.env
```

Comment:
This isn't wrong but I would strongly recommend you add it to *your* `~/.gitignore_global` too so you don't accidentally ever add it on new projects.